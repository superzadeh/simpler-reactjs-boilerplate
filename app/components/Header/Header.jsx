import React from 'react';

class Header extends React.Component { // eslint-disable-line react/prefer-stateless-function
  render() {
    return (
      <div className="header">
        Header
      </div>
    );
  }
}

export default Header;
