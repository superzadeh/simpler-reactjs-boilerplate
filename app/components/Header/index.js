import Header from './header';
import './style.scss';

export default Header;
