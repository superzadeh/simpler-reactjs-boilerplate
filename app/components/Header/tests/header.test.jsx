import React from 'react';
import { shallow } from 'enzyme';

import Header from '../header';

const child = <h1>Test</h1>;
const getElement = () => (<Header>{child}</Header>);

describe('Header', () => {
  it('should render correctly', () => {
    const header = getElement();
    expect(header).toMatchSnapshot();
  });
  it('should render an <div> tag', () => {
    const renderedComponent = shallow(getElement());
    expect(renderedComponent.type()).toEqual('div');
  });
});
