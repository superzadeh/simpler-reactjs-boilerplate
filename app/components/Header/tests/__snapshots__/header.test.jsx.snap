// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<Header>
  <h1>
    Test
  </h1>
</Header>
`;
